prowler/providers/aws/services/elasticache/elasticache_redis_cluster_backup_enabled/elasticache_redis_cluster_backup_enabled.py

```diff
@@ -13,17 +13,17 @@ def execute(self):
             report.resource_id = repl_group.id
             report.resource_arn = repl_group.arn
             report.status = "FAIL"
-            report.status_extended = f"Elasticache Cluster {repl_group.id} does not have automated snapshot backups enabled."
+            report.status_extended = f"Elasticache Redis cache cluster {repl_group.id} does not have automated snapshot backups enabled."
             if repl_group.snapshot_retention > elasticache_client.audit_config.get(
                 "minimum_snapshot_retention_period", 7
             ):
                 report.status = "PASS"
-                report.status_extended = f"Elasticache Cluster {repl_group.id} has automated snapshot backups enabled with retention period {repl_group.snapshot_retention} days."
+                report.status_extended = f"Elasticache Redis cache cluster {repl_group.id} has automated snapshot backups enabled with retention period {repl_group.snapshot_retention} days."
             else:
                 if repl_group.snapshot_retention > 0:
                     report.status = "FAIL"
                     report.check_metadata.Severity = "low"
-                    report.status_extended = f"Elasticache Cluster {repl_group.id} has automated snapshot backups enabled with retention period {repl_group.snapshot_retention} days. Recommended to increase the snapshot retention period to a minimum of 7 days."
+                    report.status_extended = f"Elasticache Redis cache cluster {repl_group.id} has automated snapshot backups enabled with retention period {repl_group.snapshot_retention} days. Recommended to increase the snapshot retention period to a minimum of 7 days."
```

Comment:
Please cover this line with a test.

Reply:
Done

Reply:
Can you double check this please? I don't see a test for this case.